backend/main.go

```diff
@@ -223,7 +223,7 @@ func main() {
 		samplingMap = map[trace.SpanKind]float64{
 			trace.SpanKindUnspecified: 1. / 1_000_000,
 			trace.SpanKindInternal:    1. / 1_000_000,
-			trace.SpanKindConsumer:    1. / 1_00,
+			trace.SpanKindConsumer:    util.ConsumerSpanSamplingRate(),
```

Comment:
🔥 I like the idea of making these env variables so we can switch them faster than needing a new PR/deploy